### DISCUSSION TITLE
Magento Version Number Confusion

### DISCUSSION THREAD

**Discussion by BrianPittVP** [OpenMage/magento-lts#2631] [Questions & Answers] [answered]
Why does the 20.x branch of OpenMage report that its version is 1.9.4.5?  Is that legacy Magento stuff?  There does not appear to be anything in the OpenMage core that reports 20.x.

**Comment by fballiano** [comment]
Where do you see that number? in the backend? because in the backend you should see 20.0.16 or something like that.

We have 2 different version numbers, the OpenMage version AND the Magento version that is based on, we kept it because some modules are checkin "if version. 1.4" or something like that

**Comment by elidrissidev** [comment, marked as answer]
`Mage::getVersion()` will give you the latest Magento version that OpenMage is based on, which is `1.9.4.5`.

`Mage::getOpenMageVersion()` will give you the actual OpenMage version you're looking for.

**Reply by BrianPittVP** [reply]
Ahh, that explains it.  Thank you!

**Reply by elidrissidev** [reply]
You're welcome.
